packages/rfw/CHANGELOG.md

```diff
@@ -1,7 +1,12 @@
 ## NEXT
 
+* Updates example compileSdkVersion to 31.
```

Comment:
Could you clarify which example app corresponds to which update?

Reply:
Sorry, I had fixed this double-NEXT locally when I uploaded, but apparently on the wrong branch as part of messing up my local branches :P